src/config/default.properties

```diff
@@ -1,3 +1,6 @@
+celery_broker=amqp://guest@localhost/%2F
```

Comment:
nb - do we want the celery broker to be locked down at all with users/permissions?